Content.Client/Crayon/CrayonDecalVisualizer.cs

```diff
@@ -19,9 +19,9 @@ public override void OnChangeData(AppearanceComponent component)
                 sprite.LayerSetState(0, state);
             }
 
-            if (component.TryGetData(CrayonVisuals.Color, out Color color))
+            if (component.TryGetData(CrayonVisuals.Color, out string color))
             {
-                sprite.LayerSetColor(0, color);
+                sprite.LayerSetColor(0, Color.FromName(color));
             }
```

Comment:
Should server just be setting Color instead?

Reply:
as we discussed in Discord, Color is not currently NetSerializable https://discord.com/channels/310555209753690112/310555209753690112/858617392648421377